src/azure-cli/azure/cli/command_modules/apim/tests/latest/api_policy.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+    This is a test policy file for Azure API Management.
+-->
+<!-- Add policies as children to the <inbound>, <outbound>, <backend>, and <on-error> elements -->
+<policies>
+    <!-- Throttle, authorize, validate, cache, or transform the requests -->
+    <inbound>
+    </inbound>
+    <!-- Control if and how the requests are forwarded to services  -->
+    <backend>
+    </backend>
+    <!-- Customize the responses -->
+    <outbound>
+    </outbound>
+    <!-- Handle exceptions and customize error responses  -->
+    <on-error>
+    </on-error>
+</policies>
```

Comment:
Suggest placing all the files referenced during testing into the  `../tests/latest/data` folder uniformly